pds_doi_core/actions/draft.py

```diff
@@ -76,7 +85,7 @@ def _resolve_input_into_list_of_names(self, input):
 
         return o_list_of_names
 
-    def _transform_pds4_label_into_osti_record(self, input_file, node, submitter, contributor_value):
+    def _transform_pds4_label_into_osti_record(self, input_file, node, submitter, contributor_value, force_flag):
```

Comment:
I think I understand more of the intention of the validate() function now.  Its job is to ensure the rule is held.  It is up to the run() function to check to see if the user wishes to do the force function if they know why the rule was violated in the first place.

Will be making the changes to the code of each action's call to validate() function.

Reply:
The force_flag is now handled inside the run() function is the user wishes to draft the same title.
The _transform_pds4_label_into_osti_record() function no longer take the force_flag parameter since it is the job of run() function to deal with the exception from the validate() function.
The validate() function now no longer take the force_flag parameter.